src/containers/shared/components/TxDetails.tsx

```diff
@@ -1,13 +1,57 @@
-import React, { Component } from 'react';
+import { ReactElement } from 'react';
 import PropTypes from 'prop-types';
 import { translate } from 'react-i18next';
 import Currency from './Currency';
 import { CURRENCY_OPTIONS, DATE_OPTIONS, ACCOUNT_FLAGS, decodeHex } from '../transactionUtils';
 import { localizeNumber, localizeDate } from '../utils';
 
-class TxDetails extends Component {
-  renderAmount = d => {
-    const { language } = this.props;
+interface Instructions {
+  owner: string;
+  sequence: number;
+  ticketSequence: number;
+  fulfillment: string;
+  finishAfter: string;
+  cancelAfter: string;
+  condition: string;
+  quorum: number;
+  max: any;
+  signers: any[];
+  domain: string;
+  email_hash: string;
+  message_key: string;
+  set_flag: string;
+  clear_flag: string;
+  key: string;
+  limit: any;
+  pair: string;
+  sourceTag: number;
+  source: string;
+  claimed: any;
+  channel_amount: number;
+  remaining: number;
+  renew: boolean;
+  close: boolean;
+  deleted: boolean;
+  gets: any;
+  pays: any;
+  price: string;
+  cancel: number;
+  convert: any;
+  amount: any;
+  destination: string;
+  partial: boolean;
+}
+
+interface Props {
+  instructions: Instructions;
+  type: string | null;
+  language: string;
+  t: (s: string) => string;
```

Comment:
What does t represent here? (I see it in almost every file, but I can't figure out what it's supposed to do)

Reply:
It's from the internationalization framework that we use: https://www.i18next.com/overview/api#t